app/src/main/res/values-pt-br/strings.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+/**
+The MIT License (MIT)
+Copyright (c) 2013 Valentin Konovalov
```

Comment:
Well, it is not mine diff - you are free to add your name here. Thanks for the PR.